webui/react/src/pages/TrialDetails.tsx

```diff
@@ -193,7 +193,9 @@ const TrialDetailsComp: React.FC = () => {
   }, [ hasCheckpointOrValidation, trial?.steps ]);
 
   const pollTrialDetails = useCallback(() => {
+    if (trialResponse.isLoading) return;
```

Comment:
blocking: could this be never false? 
could we test (maybe manually) this with say adding a delay to the trialDetail API call? 
from my testing I'm not seeing the expected behavior

![Screen Capture_select-area_20200904150040](https://user-images.githubusercontent.com/12127420/92287770-9ab5ee80-eebf-11ea-9eb9-25781b877f62.png)


```
diff --git a/webui/react/src/Dev.ts b/webui/react/src/Dev.ts
index ac3a8ee2..66e7085c 100644
--- a/webui/react/src/Dev.ts
+++ b/webui/react/src/Dev.ts
@@ -7,3 +7,9 @@ if (process.env.IS_DEV) {
   window.dev.api = Api;
   document.title = `[dev] ${document.title}`;
 }
+
+export const sleep = (ms: number) => {
+  return new Promise((resolve) => {
+    setTimeout(resolve, ms);
+  });
+};
diff --git a/webui/react/src/hooks/usePolling.ts b/webui/react/src/hooks/usePolling.ts
index cb2da09a..3299786d 100644
--- a/webui/react/src/hooks/usePolling.ts
+++ b/webui/react/src/hooks/usePolling.ts
@@ -2,7 +2,7 @@ import { useCallback, useEffect, useRef } from 'react';
 
 import { isAsyncFunction } from 'utils/data';
 
-const DEFAULT_DELAY = 5000;
+const DEFAULT_DELAY = 2000;
 
 type PollingFn = (() => Promise<void>) | (() => void);
 
diff --git a/webui/react/src/services/apiBuilder.ts b/webui/react/src/services/apiBuilder.ts
index 7f1392db..23d2074b 100644
--- a/webui/react/src/services/apiBuilder.ts
+++ b/webui/react/src/services/apiBuilder.ts
@@ -1,5 +1,6 @@
 import axios, { AxiosResponse, CancelToken, Method } from 'axios';
 
+import { sleep } from 'Dev';
 import handleError, { DaError, ErrorLevel, ErrorType, isDaError } from 'ErrorHandler';
 import { isAuthFailure } from 'services/api';
 import * as DetSwagger from 'services/api-ts-sdk';
@@ -53,11 +54,14 @@ export const processApiError = (name: string, e: Error): DaError => {
   });
 };
 
+let c = 0;
 export function generateApi<Input, Output>(api: Api<Input, Output>) {
   return async function(params: Input & { cancelToken?: CancelToken }): Promise<Output> {
+    const myC = c++;
     const httpOpts = api.httpOptions(params);
 
     try {
+      console.log('started req', myC, api.name, new Date());
       const response = await http.request({
         cancelToken: params.cancelToken,
         data: httpOpts.body,
@@ -66,6 +70,9 @@ export function generateApi<Input, Output>(api: Api<Input, Output>) {
         url: httpOpts.url as string,
       });
 
+      await sleep(10000);
+      console.log('finished req', myC, api.name, new Date());
+
       return api.postProcess ? api.postProcess(response) : response.data as Output;
     } catch (e) {
       processApiError(api.name, e);

```